### PR TITLE
test-env deployment script

### PR DESCRIPTION
**Pull request by Holymir**
- Deploys the WHBAR token and Governance contracts

- Generates 3 member accounts and send them 0.1 ethers

- Sets the required validators to be members of the Governance contract

- Prints out WHBAR, Bridge, Alice, Bob and Carol Wallets

### CODE REVIEW

**Review by failfmi — DISMISSED** [review]

File: deployment/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Deployment Scripts
```
Comment (Daniel-K-Ivanov) [inline deployment/README.md:1]: I think that it would be better if this is placed in the main readme file

File: deployment/deploy.js [hunk 2]
```diff
@@ -18,14 +18,20 @@ const deploy = async (network, secret) => {
         deployer = new etherlime.InfuraPrivateKeyDeployer(secret, network, INFURA_PROVIDER);
     }
 
-    whbarInstance = await deployer.deploy(WHBAR, {}, "Name", "Symbol", 8);
+    whbarInstance = await deployer.deploy(WHBAR, {}, "Wrapper HBAR", "WHBAR", 8);
```
Comment (Daniel-K-Ivanov) [inline deployment/deploy.js:21]: nit: "Wrapped HBAR"

File: deployment/testnet-env-deployments.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+const deploy = async (network, secret) => {
+    let deployer;
+
+    if (!network) {
+        deployer = new etherlime.EtherlimeGanacheDeployer();
+    } else {
+        deployer = new etherlime.InfuraPrivateKeyDeployer(secret, network, INFURA_PROVIDER);
+    }
+
+    whbarInstance = await deployer.deploy(WHBAR, {}, "Wrapper HBAR", "WHBAR", 8);
```
Comment (Daniel-K-Ivanov) [inline deployment/testnet-env-deployments.js:20]: nit: `Wrapped`

**Review by Daniel-K-Ivanov — CHANGES_REQUESTED** [review]

**Review by Daniel-K-Ivanov — DISMISSED** [review]
Do we even need the `./scripts/hedera-deploy.js` file and the corresponding documentation in the main README.md file? I think not

**Review by Daniel-K-Ivanov — APPROVED** [review]

**Review by failfmi — APPROVED** [review]

**Pull request merged**